style_editor.html

```diff
@@ -7355,16 +7401,18 @@
   <td valign=top width=50%>
     <canvas id="canvas_id" width=256px height=256px style="width:256px; height:256px;"> </canvas>
     <br>
-    <input id=POWER_BUTTON name=on type=button value="Power" onclick="ClickPower();" >
-    <input name=clash type=button value="Clash" onclick="AddClash();" >
-    <input name=stab type=button value="Stab" onclick="AddStab();" >
-    <input name=blast type=button value="Blast" onclick="AddBlast()"; >
-    <input name=force type=button value="Force" onclick="AddForce()"; >
-    <input name=boot type=button value="Boot" onclick="AddBoot()"; >
-    <input name=newfont type=button value="NewFont" onclick="AddNewfont()"; >
-    <input name=preon type=button value="Preon" onclick="AddPreon()"; >
-
-    <select name=lockup id=LOCKUP onchange='OnLockupChange()'>
+    <input id="PREON_BUTTON" name="preon" type="button" value="Preon" onclick="AddPreon()">
+    <input id="POWER_BUTTON" name="on" type="button" value="Power" onclick="ClickPower();">
+    <input id="NEWFONT_BUTTON" name="newfont" type="button" value="NewFont" onClick="AddNewfont()">
+    <input id="CLASH_BUTTON" disabled="true" name="clash" type="button" value="Clash" onclick="AddClash();">
+    <input id="STAB_BUTTON" disabled="true" name="stab" type="button" value="Stab" onclick="AddStab();">
+    <input id="BLAST_BUTTON" disabled="true" name="blast" type="button" value="Blast" onclick="AddBlast()">
+    <input id="FORCE_BUTTON"  disabled="true" name="force" type="button" value="Force" onclick="AddForce()">
+    <input id="BOOT_BUTTON" name=boot type=button value="Boot" onclick="AddBoot()"; >
+    <input id="BATTERY_BUTTON" name="battery" type="button" value="Battery Level" onClick="AddBattery()">
+    <input id="POWERSAVE_BUTTON" disabled="true" name="powersave" type="button" value="Powersave" onClick="AddPowersave()">	 
```

Comment:
What if we hid the buttons instead of just disabling?  It would free up space if we just did "display: none" based on STATE_ON

Reply:
Might work. Hopefully it's not too confusing.

Reply:
I'll play around will it, I figure in normal use you can't trigger blast, clash, force, lockup, etc while the saber is off anyway.  Maybe have the editor always default to STATE_ON since most actions are while on anyway, then when the saber is turned off and those buttons go away it makes more sense?